### DISCUSSION TITLE
Run clean_zone in Home Assistant?

### DISCUSSION THREAD

**Discussion by kaischlotmann** [jmzg93/koboldVR7Integration#3] [Q&A]
Hello,
I am newbie and generally got this great integration work!
But how to call the service? How is syntax in HA? Do I use a script?
Does anyone have an example for me?
map and zones are shown in the integration properly.
Thanks
Kai

**Comment by jmzg93** [comment]
> Hello,
> I am newbie and generally got this great integration work!
> But how to call the service? How is syntax in HA? Do I use a script?
> Does anyone have an example for me?
> map and zones are shown in the integration properly.
> Thanks
> Kai

Hi! Sorry for the delay in replying, I didn't miss any notifications about this :(
Currently, you can't select which zone to run. The integration does detect them, but they aren't selectable. Sorry, I did this in my spare time, and since it was my first and only Home Assistant integration, when I saw that it worked with all the zones, I stopped moving forward with it.
I'll try to add more functionality in the future. I hope you understand. Thanks, and best regards.

**Reply by jmzg93** [reply]
Hello @kaischlotmann !! This part should be working now. Let me know if there's anything I can do. Best regards.
